planner/core/plan_cost_test.go

```diff
@@ -119,12 +120,13 @@ func TestScanOnSmallTable(t *testing.T) {
 	}
 
 	rs := tk.MustQuery("explain select * from t").Rows()
-	useTiKVScan := true
+	useTiKVScan := false
 	for _, r := range rs {
+		fmt.Println(">>> ", r)
```

Comment:
Remove this part.
BTW, why we change the set order for `useTiKVScan`.

Reply:
My mistake. It's removed.